src/javascripts/zendesk-api/config.js

```diff
@@ -60,6 +77,13 @@ var config = module.exports = {
       this.ajax('activatedLocales')
         .done(data => {
           this.activatedLocalesDone(data);
+          this.ajax('activatedExtraLocales')
+          .done(data => {
+            this.activatedExtraLocalesDone(data);
+          })
+          .fail(xhr => {
+            this.activatedLocalesFail(xhr);
+          });
```

Comment:
we probably need a comment here due to the nested chaining ...

Reply:
done